### ISSUE TITLE
Browser quits right after opened with ElementExplorer

### ISSUE THREAD

**Issue by PMarian0** [angular/protractor#2228]
When I start the elementExplorer with Protractor, somehow the browser (either Chrome or Firefox) appear onscreen for a split second, and then shuts down.

 [Here is a short video] (http://screencast.com/t/cZjnHJVTZiT) where I use --troubleshoot. (I'm on W7 x64) using
```node
protractor --elementExplorer http://www.angularjs.org
```

Console reaches 'listening on port ####' and hangs if I use
```node
node ./bin/elementexplorer.js http://www.angularjs.org
```

 I know about the issue in #1890; but as I cannot start and take control of the browser, I think this is a different issue.
 Will try downgrading Protractor/Node later on today and will post if anything changes.

Updates:
_Note: Downgraded Node by removing it and reinstalling from scratch._

Update  # | Node | NPM | Protractor | Issue
 -------- | ----- | ------ | ------------- | --------
0 | 0.12.3 | 2.9.1 | 2.1.0 | Browser closes itself
1 | 0.12.3 | 2.9.1 | 1.6.1 | Browser closes itself
2 | 0.10.30 | 1.4.21 | 1.7.0 | Browser closes after console input
3 | 0.10.30 | 1.4.21 | 1.6.1 | ECONNREFUSED
4 | 0.10.30 | 1.4.21 | 1.6.1 | Work? (see below)
5 | 0.12.4 | 2.10.1 | 2.10.1 | Work? (see below) + small change + #1970 

------------------------------------------------------------------------------------------------

##### 1st Update: #####
 Seems to be an issue with W7/W8. A test on Ubuntu 14.10 with 
 - Node: v0.12.3
 - Npm: 2.9.1
 - Protractor: 2.0.0
seems to be working as expected, somehow.

------------------------------------------------------------------------------------------------
 
##### 3rd Update: #####
For the 3d row, I had to use the _node ./protractor/bin[...]_ command as protractor --elementExplorer is not available

------------------------------------------------------------------------------------------------

##### 4th update ####
Found out via 
```node
npm ls -depth=0 I have protractor@2.0.0;
```
And by

```node
npm ls -g -depth=0 I have protractor@1.6.1;
```
Running _node ./protractor/bin[...]_ seems to work OK now.
Weird thin is if I run the command above, it tells me to better use protractor --elementExplorer ...which is not available. (this because I have the 2.0.0, I guess)

----------------------------------------------------------------------------------------------------------

##### 5th Update: #####

 Entered full update mode. Seems to work, but:
 - protractor --elementExplorer will shut down browser after a jiffy onscreen.
 - If I enter _node ./protractor/bin[...]_ + urL, console will get to 'Listenting to port....' and hang. If I don't, it opens a browser.
 - Commands not sent due to console cutting first letter.
 - If I send, for instance, *abrowser.get('http://....');* , works. As long as I add the char that will be 'eaten' by the console, I'm saved.
 - Sending url directly with no _http://_ prefix will return timeout (might be expected behaviour?)

---------------------------------------------------------------------------------------------------------------
 Will continue to investigate around jic.

**Comment by hankduan** [comment]
Sorry I'm not certain what exactly the issue is at the moment. Can you run protractor 2.1.0 with node 0.10.30, and let me know what you see going wrong?

**Comment by PMarian0** [comment]
Sorry I'm having a really loaded month, will try today or tomorrow when I have some free time.

**Comment by PMarian0** [comment]
@hankduan works exactly like row no.4 of the table

**Comment by sipanda84** [comment]
I've experienced the same issue but I think it's protractor issue. 
This is what I have on my settings:

NodeJs: 0.10.30 (don't use latest 0.12.0)
Selenium-server: 2.46.0 (latest one)
Selenium-webdriver: 2.46.1 (latest one)
Selenium-server-standalone: 2.46.0 (latest one)
Chromedriver: 2.16.0 (latest one)
Protractor: 1.8.0

I have tried protractor 2.0.0 and 2.1.0. The test would run but running the elementExplorer would shutdown the browser as described in the ticket.
